<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>241</x>
      <y>61</y>
      <width>80</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>241</x>
      <y>14</y>
      <width>491</width>
      <height>301</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="mainLabel">
       <property name="text">
        <string>    DBLP科学文献管理系统</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pushButtonSearch">
       <property name="text">
        <string>基本搜索</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="pushButtonHot">
       <property name="text">
        <string>热点排序</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="pushButtonCluster">
       <property name="text">
        <string>聚团分析</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="pushButtonAuthor">
       <property name="text">
        <string>作者排序</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDBLP">
    <property name="title">
     <string>DBLP</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionexit"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuInit">
    <property name="title">
     <string>Init</string>
    </property>
    <addaction name="actionInit"/>
   </widget>
   <addaction name="menuDBLP"/>
   <addaction name="menuInit"/>
  </widget>
  <action name="actionexit">
   <property name="text">
    <string>exit</string>
   </property>
  </action>
  <action name="actionInit">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Init</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
